### PR TITLE
BIP141: Sigop clarification

### PR DESCRIPTION
**Pull request by jl2012**

### CODE REVIEW

File: bip-0141.mediawiki [hunk 1]
```diff
@@ -118,7 +118,7 @@ Sigops in the current pubkey script, signature script, and P2SH check script are
 The sigop limit is likewise quadrupled to ≤ 80,000.
 
 In addition, opcodes within the witness program are counted identical to as previously within the P2SH check script.
-That is, CHECKSIG is counted as only 1 sigop in a witness program, despite P2SH check script's CHECKSIG becoming 4 sigops with the forementioned change.
+That is, CHECKSIG in a witness program is counted as only 1 sigop, and m-of-n CHECKMULTISIG in a witness program is counted as n sigop. This rule applies to both native witness program and P2SH witness program.
```
Comment (luke-jr) [inline bip-0141.mediawiki]: This removes the contrast with the quadrupling of the current scripts

Comment (jl2012) [inline bip-0141.mediawiki]: it sounds a bit cumbersome, as it already mentions "is counted as only 1 sigop".

I also want to make it clear that witness program in P2SH is also counted without quadrupling.

Comment (luke-jr) [inline bip-0141.mediawiki]: Also, m-of-n CHECKMULTISIG is only n sigop in some circumstances, not all - if the preceding opcode is not a constant between 1 and 20, it unconditionally counts as 20 sigops. (I assume this hasn't changed for segwit?)

**Comment by jl2012** [comment]
looks better now?

**Comment by luke-jr** [comment]
Yeah

**Pull request merged**